xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -241,7 +241,16 @@
       <Docs>
         <param name="eventSourceName">The name to apply to the event source. Must not be <see langword="null" />.</param>
         <summary>Creates a new instance of the <see cref="T:System.Diagnostics.Tracing.EventSource" /> class with the specified name.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+  ## Remarks
+
+  Constructing an EventSource via this constructor to specify its name while not specifying a name via <xref:System.Diagnostics.Tracing.EventSourceAttribute> can result in the `EventSource.GetGuid(Type)` call to return a different GUID from the <xref:System.Diagnostics.Tracing.EventSource.Guid> property.
```

Comment:
This wording is a little bit confusing and abstract.

Can we give guidance:
1. Use the constructor that doesn't create this issue.
2. Word it suggestively: If using this, please make sure to use the attribute **ref** using the same name as the parameter to this constructor. Otherwise, there will be inconsistent GUIDs between the *property* and the one returned by *GetGuid*. The property will be the authoritative identifier while using ETW.